lib/Conversion/PolyToStandard/PolyToStandard.cpp

```diff
@@ -41,7 +41,9 @@ struct ConvertAdd : public OpConversionPattern<AddOp> {
   LogicalResult matchAndRewrite(
       AddOp op, OpAdaptor adaptor,
       ConversionPatternRewriter &rewriter) const override {
-    // TODO: implement
+    arith::AddIOp addOp = rewriter.create<arith::AddIOp>(
+        op.getLoc(), adaptor.getLhs(), adaptor.getRhs());
+    rewriter.replaceOp(op.getOperation(), {addOp});
```

Comment:
`{addOp}` will trigger ambiguous error in LLVM 20, so this can be implemented in this way:
```c
rewriter.replaceOp(op.getOperation(), addOp.getOperation());
```